WORKSPACE

```diff
@@ -113,9 +120,16 @@ new_git_repository(
     remote = "https://github.com/googleapis/googleapis.git",
 )
 
+# Dependency for org_golang_google_grpc
+new_go_repository(
```

Comment:
curious: are there any broader concerns with having this dep and the other, different genproto deps all in our WORKSPACE?